src/polyfill/binding.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  /* jshint expr: true */
+  create (opts) {
+    var elem = document.createElement(opts.extends || opts.id);
+    opts.extends && elem.setAttribute('is', opts.id);
+    return elem;
```

Comment:
Say we had

```js
skate('x-foo', …);
skate('bar', { extends: 'x-foo', … });
```

does this creation order (1. `createElement`, 2. add the `is` attr) risk running the `x-foo` `created` callback when if you had done `….innerHTML = '<x-foo is="bar"></x-foo>'` you wouldn't have?